### DISCUSSION TITLE
sddc的未使用函数修剪功能为啥没了

### DISCUSSION THREAD

**Discussion by wc7086** [github0null/eide#295] [Q&A] [answered]
https://discuss.em-ide.com/d/336-new-sdcc

**Comment by github0null** [comment, marked as answer]
这个功能需要解析 asm 来实现修剪，

但我还没找到 sdcc asm 的完整语法文档，因此语法支持不完善，导致这个功能有 bug，某些时候会编译失败

先暂时删除了这个功能，后面有新的实现方法了，会加上去的
